src/CattyTests/CatrobatLanguage0.992/XMLSerializerTests/BlackBox Tests/XMLSerializerBlackBoxTests.swift

```diff
@@ -26,106 +26,102 @@ import XCTest
     }
 
     func testFlappy30() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Flappy_v3.0_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Flappy_v3.0_0992")
     }
 
     func testGossipGirl() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Gossip_Girl_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Gossip_Girl_0992")
     }
 
     func testMemory() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Memory_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Memory_0992")
     }
 
     func testMinecraftWorkInProgress() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Minecraft_Work_In_Progress_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Minecraft_Work_In_Progress_0992")
     }
 
     func testMinions() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Minions__0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Minions__0992")
     }
 
     func testPongStarter() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Pong_Starter_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Pong_Starter_0992")
     }
 
     func testRockPaperScissors() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Rock_paper_scissors_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Rock_paper_scissors_0992")
     }
 
     func testTicTacToeMaster() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Tic_Tac_Toe_Master_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Tic_Tac_Toe_Master_0992")
     }
 
     func testWordBalloonDemo() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Word_balloon_demo_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Word_balloon_demo_0992")
     }
 
     func testXRayPhone() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "X_Ray_phone_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "X_Ray_phone_0992")
     }
 
     func testGalaxyWar() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Galaxy_War_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Galaxy_War_0992")
     }
 
     func testSkydivingSteve() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Skydiving_Steve_0991")
+        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Skydiving_Steve_0992")
     }
 
     func testPythagoreanTheorem() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "Pythagorean_Theorem_0991")
-    }
-
-    func testValidProjectAllBricks() {
-        self.testParseXMLAndSerializeProjectAndCompareXML(xmlFile: "ValidProjectAllBricks0991")
```

Comment:
I think the `testValidProjectAllBricks` is now missing.

Reply:
Yes, because we don't have the XML for ValidProjectAllBricks i.e ValidProjectAllBricks0992